celestia/peers.txt

```diff
@@ -1 +1 @@
-
+398a17a83f8af55157ecb33ea81d4a9d5b9953e0@178.63.76.62:26656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`